### PR TITLE
Set up a test suite

### PR DESCRIPTION
**Pull request by pierrealixt**

### CODE REVIEW

File: deployment/Makefile [hunk 1]
```diff
@@ -338,19 +338,9 @@ build-devweb: db
...
-	@echo "PEP8 issues"
-	@echo "-----------"
-	@pep8 --version
-	@pep8 --repeat --ignore=E203,E121,E122,E123,E124,E125,E126,E127,E128,E402  --exclude='../django_project/.pycharm_helpers','../django_project/*/migrations/','../django_project/*/urls.py','../django_project/core/settings/secret.py' ../django_project || true
-
 flake8:
-	@echo "Check flake 8"
-	@flake8 ../django_project
+	@docker-compose exec uwsgi flake8 --version
+	@docker-compose exec uwsgi flake8 --ignore=E203,E121,E122,E123,E124,E125,E126,E127,E128,E402 --exclude='../django_project/.pycharm_helpers','../django_project/*/migrations/','../django_project/*/urls.py','../django_project/core/settings/secret.py' ../django_project || true
```
Comment (lucernae) [inline deployment/Makefile]: I think it's probably better to put this in a 'tox.ini' file or '.flake8' file

**Review by lucernae — APPROVED** [review]
LGTM

**Comment by pierrealixt** [comment]
`make test` runs the test suite.
`make flake8` checks the code.

File: deployment/Makefile (hunk 1 above)
Comment (lucernae) [reply deployment/Makefile]: Ah, we already have setup.cfg files. I think you can put flake config there.

**Pull request merged**